src/main/java/appeng/util/Lazy.java

```diff
@@ -16,17 +16,29 @@
  * along with Applied Energistics 2.  If not, see <http://www.gnu.org/licenses/lgpl>.
  */
 
-package appeng.container.slot;
+package appeng.util;
 
 
-import net.minecraft.inventory.IInventory;
+import java.util.function.Supplier;
 
 
-public class QuartzKnifeOutput extends SlotOutput
+public class Lazy<T> implements Supplier<T>
 {
+	private final Supplier<T> supplier;
+	private T instance = null;
 
-	public QuartzKnifeOutput( final IInventory a, final int b, final int c, final int d, final int i )
+	public Lazy( final Supplier<T> supplier )
```

Comment:
ContainCellWorkbench.setupConfig is called in the constructor of the superclass, therefore the class is not fully initalized there. In the past there was a custom wrapper there which implicitly made it lazy.